lib/ex_unit/lib/ex_unit/runner.ex

```diff
@@ -188,11 +188,34 @@ defmodule ExUnit.Runner do
       {:error, %{test_case | state: failed}}
   end
 
-  defp run_test(config, test, context) do
+  defp run_test(true, config, test, context) do
+    run_test(:stdio, config, test, context)
+  end
+
+  defp run_test(false, config, test, context) do
+    spawn_test(config, test, context)
+  end
+
+  defp run_test(device, config, test, context) do
+    try do
+      logged =
+        ExUnit.CaptureLog.capture_log(device, fn ->
+          send self(), {:capture, spawn_test(config, test, context)}
+        end)
+      receive do
+        {:capture, test} -> %{test | logs: logged}
+      end
+    catch
+      :exit, :noproc -> %{test | state: {:skip, :no_logger}}
```

Comment:
We don't want to skip it. We should either tag it as an error or mark it as invalid. I think I prefer an error, fits better with everything else. :)